Shared/Utils.swift

```diff
@@ -0,0 +1,50 @@
+//
+//  Utils.swift
+//  Client
+//
+//  Created by Atul Aggarwal on 8/8/15.
+//  Copyright (c) 2015 Mozilla. All rights reserved.
+//
+
+import AVFoundation
+import UIKit
+
+public func getUserAgent() -> String {
```

Comment:
Using bare functions like this for very specific functionality (as opposed to combining utilities) muddles the top-level namespace. Take a look at `DeviceInfo` for the pattern to follow: name this file `UserAgent.swift`, introduce a class here named `UserAgent`, and call this static method `defaultUserAgent()`.